.github/workflows/reusable_sanitizers.yml

```diff
@@ -77,7 +77,6 @@ jobs:
         ASAN_OPTIONS: allocator_may_return_null=1
         TSAN_OPTIONS: allocator_may_return_null=1
       run: |
-        ${{ matrix.compiler.cxx == 'icpx' && '. /opt/intel/oneapi/setvars.sh' || true }}
         ctest --output-on-failure
```

Comment:
Why is `LD_LIBRARY_PATH=${{env.BUILD_DIR}}/lib/` not needed like it is above?

Reply:
@KFilipek please check if setting LD_LIBRARY_PATH is still needed if we don't source oneAPI env vars

Reply:
It's still needed by shared library, this case above is compiled statically with the UMF, so there is no need to pass shared library.